src/components/compound/DlSearches/DlSmartSearch/components/DlSuggestionsDropdown.vue

```diff
@@ -44,7 +44,7 @@ export default defineComponent({
     },
     model: {
         prop: 'modelValue',
-        event: 'update:modelValue'
+        event: 'update:model-value'
```

Comment:
Nice catch ! does this still work on vue3 normally? 
i believe it shoudl since we are explicitly saying what the event should be

Reply:
Yeah, it works well on vue 3 as well